expansion/src/test/java/no/unit/nva/expansion/model/ExpandedDataEntryTest.java

```diff
@@ -66,15 +74,22 @@ public void setup() {
         this.resourceExpansionService =
             new ResourceExpansionServiceImpl(resourceService, ticketService);
     }
-    
+
     @Test
     void shouldReturnExpandedResourceWithoutLossOfInformation() throws JsonProcessingException {
         var publication = createPublicationWithoutDoi();
+
+        publication.getEntityDescription().getContributors().forEach(
+            cont ->
+                cont.getAffiliations().remove(0)
```

Comment:
What is going on here? What if getAffiliations() has no elements? Probably not a problem in this test, but is it a reason why you always delete the first affiliation?

Reply:
Had some trouble when it came to mocking. Found a better solution. Please see reviewed version. 


In case you want insight on what the problem was:
This test assumes we're not expanding any URI's by not providing any URI's in fields that should be expanded. Now that we're expanding affiliations this test fails (especially since the mockdata points to example.org/somestring which returns html, so we get a parsing error). 

The approach i tried here removes the affiliation-field from the testdata entirely, making the test pass. This is not optimal since we're not testing for changes in affiliations. I.E. if the framing for affiliations is wrong the test will still pass.

My new solution will make sure we return an empty JSON (meaning we're still not expanding) instead. This way the test will still test everything it's supposed to test.